altos-rust/altos-core/libs/heap/free_list_allocator/src/test.rs

```diff
@@ -131,9 +131,9 @@ impl TestFreeList {
 
     // Helper function to check that every block in the list satisfies some condition
     // Returns false if the condition returns false for any node
-    pub fn each_free_block_satisfies<F: Fn(&free_list::BlockHeader) -> bool>
-        (&self, condition: F) -> bool {
-
+    pub fn each_free_block_satisfies<F>(&self, condition: F) -> bool
+        where F: Fn(&free_list::BlockHeader) -> bool
+    {
```

Comment:
Hm, yeah, we want to keep it the same. Personally I like the bracket on it's own line if the function signature extends across multiple lines so it's clear where the signature ends and the function body begins. Any objections to adopting that as style?